src/test/resources/features/CreateJourneyQA.feature

```diff
@@ -0,0 +1,228 @@
+    And I choose a repayment Type
+    When I click on the "Continue" button
+    Then I am on "Upload the multiple entries spreadsheet - Apply for repayment of import duty and import VAT - GOV.UK" page
+    When I click on Choose file button and add the "second" file
+    When I click on the "Continue" button
+    Then I am on "Enter the details of the oldest entry - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I enter declaration details- EPU: "345" and Entry No: "123456t"
+    And I enter entryDate- Day: "08", Month: "12" and Year: "2019"
+    When I click on the "Continue" button
+    Then I am on "What does your application relate to? - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I choose an application Reason
+    When I click on the "Continue" button
+    Then I am on "Tell us why the overpayment happened - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I enter goods details- Details: "Shoes Jeans Jackets"
+    When I click on the "Continue" button
+    Then I am on "What do you want to reclaim? - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I choose customsDuty importVAT anotherTypeofDuty options
+    When I click on the "Continue" button
+    Then I am on "Customs Duty overpayment - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I enter customs Duty paid- CustomDuty: "1000.00"
+    And I enter customs Duty should have paid to HMRC- CustomsDutyBeenPaid: "0.00"
+    When I click on the "Continue" button
+    Then I am on "Import VAT overpayment - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I enter import vat paid- VatPaid: "1500.00"
+    And I enter import vat been Paid to HMRC- VatBeenPaid: "0.00"
+    When I click on the "Continue" button
+    Then I am on "Overpayment of other duties - Apply for repayment of import duty and import VAT - GOV.UK" page
+    And I enter other custom Duty paid- OtherCustDutyPaid: "1250.00"
+    And I enter other custom Duty been paid to HMRC- OtherCustDutyBeenPaid: "0.00"
+    When I click on the "Continue" button
+    Then I am on "Repayment amount summary - Apply for repayment of import duty and import VAT - GOV.UK" page
+    When I click on the "Continue" button
+    Then I am on "Required supporting documents - Apply for repayment of import duty and import VAT - GOV.UK" page
+    When I click on the "Continue" button
+    Then I am on "Upload your supporting documentation - Apply for repayment of import duty and import VAT - GOV.UK" page
+    When I click on Choose file button and add the "first" file
+    And I wait for the file to be uploaded
+    Then I should see first uploaded doc "JPEGImage.jpg" on upload page
+    When I click the file continue button
+
+    # Un-comment if EORI integration is disabled in QA
```

Comment:
CLARIFICATION: Is there other magic in the acceptance test structure, that makes sure the Enrolment and EORINumber on the users session, before here?  If not, won't we fail login in QA?

Reply:
That bit has already been added - first attempt at fixing ATs in QA